index.js

```diff
@@ -23,7 +23,22 @@ module.exports = {
     });
   },
 
-  included: function(app) {
+  included: function() {
+    let app;
+
+    // If the addon has the _findHost() method (in ember-cli >= 2.7.0), we'll just
+    // use that.
+    if (typeof this._findHost === 'function') {
```

Comment:
Lifted from ember-cli-mirage, handles backwards compatibility with older versions of ember-cli.

Reply:
Is this add-on still useful?

Reply:
Yeah this addon supports inline let, and ember didn't ship that yet. https://github.com/thefrontside/ember-let/issues/49#issuecomment-430285547

I thought the same thing

Reply:
🤔